show/interfaces/__init__.py

```diff
@@ -467,8 +476,48 @@ def pm(interfacename, namespace, verbose):
     if namespace is not None:
         cmd += ['-n', str(namespace)]
 
+    cmd += ['-s', window_size, '-i', str(window_index)]
     clicommon.run_command(cmd, display_cmd=verbose)
 
+@pm.command(help="Display current PM data for the specified window size")
+@click.argument('window_size', type=click.Choice(['60sec', '15min', '24hrs']))
+@click.argument('interfacename', required=False)
```

Comment:
please capture the CLI sub command option and help text in each level from "show int trans pm" in the unit test.

Reply:
added